pkg/ccip/bindings/feequoter/fee_quoter.go

```diff
@@ -44,6 +44,18 @@ type DestChainConfig struct {
 	NetworkFeeUsdCents                uint32 `tlb:"## 32"`
 }
 
+type DestChainConfig struct {
+	ChainConfig             ChainConfig      `tlb:"."`
+	USDPerUnitGas           *cell.Cell       `tlb:"^"`
+	TokenTransferFeeConfigs *cell.Dictionary `tlb:"dict 267"`
```

Comment:
Since we aren't exposing these two fields you can revert changes to the bindings here